assets/js/data/validation/selectors.ts

```diff
@@ -3,17 +3,33 @@
  */
 import type { State } from './reducers';
 
-export const getValidationError = ( state: State ) => {
-	return ( errorId: string ) => state[ errorId ];
-};
-export const getValidationErrorId = ( state: State ) => {
-	return ( errorId: string ) => {
-		if ( ! state.hasOwnProperty( errorId ) || state[ errorId ].hidden ) {
-			return;
-		}
-		return `validate-error-${ errorId }`;
-	};
+/**
+ * Gets a validation error by ID.
+ *
+ * @param  state   The current state.
+ * @param  errorId { string } The error ID.
+ */
```

Comment:
When doing a global search for `@param`, it appears that we usually mention the `type` before the `name of the variable`, and that we're also mention the `@return` type, for example here: https://github.com/woocommerce/woocommerce-blocks/blob/296ccef838cf90a47bba412c9742b92c1658f070/assets/js/data/cart/selectors.ts#L19-L24

For consistency reasons, I wonder if we should change this part

```js
/**
 * Gets a validation error by ID.
 *
 * @param  state   The current state.
 * @param  errorId { string } The error ID.
 */
```

to that part

```js
/**
 * Gets a validation error by ID.
 *
 * @param { CartState } state The current state.
 * @param { string } errorId The error ID.
 * @param { string } The validation error.
 */
```